diff --git a/node_modules/@sitecore-jss/sitecore-jss/dist/cjs/layout/content-styles.js b/node_modules/@sitecore-jss/sitecore-jss/dist/cjs/layout/content-styles.js
index 2e75989..548f3d9 100644
--- a/node_modules/@sitecore-jss/sitecore-jss/dist/cjs/layout/content-styles.js
+++ b/node_modules/@sitecore-jss/sitecore-jss/dist/cjs/layout/content-styles.js
@@ -39,6 +39,7 @@ exports.traversePlaceholder = traversePlaceholder;
 const traverseField = (field, config) => {
     if (!field || config.loadStyles)
         return;
+    if (typeof field === 'string' && field) return;
     if ('editable' in field && field.editable) {
         config.loadStyles = CLASS_REGEXP.test(field.editable);
     }
